.travis.yml

```diff
@@ -7,6 +7,7 @@ node_js:
   - '4'
 
 before_install: yarn global add greenkeeper-lockfile@1
+install: yarn install --ignore-engines
```

Comment:
this is needed as a workaround for 
![screen shot 19](https://user-images.githubusercontent.com/22868432/34462673-a7b1a170-ee7b-11e7-83d1-13b0494b6bf5.png)
![screen shot 23](https://user-images.githubusercontent.com/22868432/34462675-bcf9dd22-ee7b-11e7-8ed9-154fe70b8f60.png)

Reply:
This will be required for `jest` as well.

Reply:
Yes. I also suggested this https://github.com/okonet/lint-staged/pull/367#issuecomment-354605443